src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+    private double hypotenuse;
```

Comment:
The hypotenuse field is not necessary for a RightTriangle class since it can be calculated from the other two legs. Consider removing this field to simplify the class design.